x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/dataframe/DataFrameAnalyticsTask.java

```diff
@@ -154,11 +158,17 @@ private void cancelReindexingTask(String reason, TimeValue timeout) {
 
     public void updateState(DataFrameAnalyticsState state, @Nullable String reason) {
         DataFrameAnalyticsTaskState newTaskState = new DataFrameAnalyticsTaskState(state, getAllocationId(), reason);
-        updatePersistentTaskState(newTaskState, ActionListener.wrap(
-            updatedTask -> LOGGER.info("[{}] Successfully update task state to [{}]", getParams().getId(), state),
-            e -> LOGGER.error(new ParameterizedMessage("[{}] Could not update task state to [{}] with reason [{}]",
-                getParams().getId(), state, reason), e)
-        ));
+        updatePersistentTaskState(
+            newTaskState,
+            ActionListener.wrap(
+                updatedTask -> {
+                    auditor.info(getParams().getId(), Messages.getMessage(Messages.DATA_FRAME_ANALYTICS_AUDIT_UPDATED_STATE, state));
```

Comment:
Given we audit state changes, I wonder if some audit messages are a bit redundant. Could you paste all messages in a happy run of a job please?

Reply:
Here are the messages from the integration test. Note that there will be more of them once I add messages for phases:
```
            "Created analytics with analysis type [regression]",
            "Estimated memory usage for this analytics to be",
            "Started analytics",
            "Creating destination index [regression_only_training_data_and_training_percent_is_fifty_source_index_results]",
            "Finished reindexing to destination index [regression_only_training_data_and_training_percent_is_fifty_source_index_results]",
            "Finished analysis"
```

Reply:
OK, this looks pretty good.

Reply:
Hold on, what about the `...updated state..` messages? Those are the ones I'm worried about.